### ISSUE TITLE
训练qwen模型出错

### ISSUE THREAD

**Issue by latte-11** [shibing624/pycorrector#561]
模型被加载到CPU而输入数据在GPU
2025-04-13 14:42:27.105 | INFO     | pycorrector.gpt.gpt_model:load_peft_model:192 - Loaded peft model from ./outputs-qwen-1.5b-demo/
2025-04-13 14:42:27.105 | DEBUG    | pycorrector.gpt.gpt_corrector:__init__:35 - Loaded model: shibing624/Qwen2.5-1.5B-Instruct, spend: 2.417 s.
Generating outputs:   0%|                                                                                                                                                                                  | 0/1 [00:00<?, ?it/s]The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
C:\Users\24069\.conda\envs\yuyi\lib\site-packages\transformers\generation\utils.py:2097: UserWarning: You are calling .generate() with the `input_ids` being on a device type different than your model's device. `input_ids` is on cuda, whereas the model is on cpu. You may experience unexpected behaviors or slower generation. Please make sure that you have put `input_ids` to the correct device by calling for example input_ids = input_ids.to('cpu') before running `.generate()`.

**Comment by shibing624** [comment]
训练时，把use_cuda=True，cuda_device=0指定下。

**Comment by latte-11** [comment]
程序在训练后报错，训练过程中显示了 Device: cuda，是不是这个报错不用管

**Comment by shibing624** [comment]
可能是显存不足，先使用qwen2.5-1.5b的模型试试

**Comment by latte-11** [comment]
显存8g够吗

**Comment by shibing624** [comment]
8g试试0.5b的模型，一般需要24gb显存。

**Comment by latte-11** [comment]
好的，我试试，谢谢了
